tools/batch/submit-job.py

```diff
@@ -87,14 +84,27 @@ def nowInMillis():
     endTime = long(total_seconds(datetime.utcnow() - datetime(1970, 1, 1))) * 1000
     return endTime
 
+job_definitions = {
+    'g4dn.4x': 'gluon-nlp-1-jobs:5',
+    'g4dn.8x': 'gluon-nlp-1-jobs:4',
+    'g4dn.12x': 'gluon-nlp-1-4gpu-jobs:1',
+    'g4dn.16x': 'gluon-nlp-1-jobs:3',
+    'p3.2x': 'gluon-nlp-1-jobs:5',
+    'p3.8x': 'gluon-nlp-1-4gpu-jobs:2',
+    'p3.16x': 'gluon-nlp-1-8gpu-jobs:1',
+    'p3dn.24x': 'gluon-nlp-1-8gpu-jobs:2'
+}
```

Comment:
These states are in gluonnlp-dev account. If we want to add storage or other peripherals we can edit them.